addons/website/static/src/snippets/s_popup/options.js

```diff
@@ -33,13 +33,17 @@ options.registry.SnippetPopup = options.Class.extend({
      * @override
      */
     onTargetShow: async function () {
-        this.$target.removeClass('d-none');
+        this.$target.addClass('d-block');
+        this.$target.css('opacity', '1');
+        $('body').addClass('modal-open');
     },
     /**
      * @override
      */
     onTargetHide: async function () {
-        this.$target.addClass('d-none');
+        this.$target.removeClass('d-block');
+        this.$target.css('opacity', '');
+        $('body').removeClass('modal-open');
```

Comment:
Seems really strange, can't you use `this.$target.modal('show')` ? If those are now always standard bootstrap modal